ui/src/app/views/workspace/component.html

```diff
@@ -5,7 +5,7 @@
     <h3>CDR Version</h3>
     <div class="indented">
       <!-- TODO: Add CDR Version Info - blrubenstein -->
-      <p>CDR version info goes here.</p>
+      <p id="cdr-text">CDR version info goes here.</p>
```

Comment:
Can we stick to class attributes rather than IDs for tests (or, alternatively, `data-test-id` attributes like those used in FireCloud)? Even though not strictly enforced, my understanding is that IDs should be globally unique across the page, which would be extra work to ensure.